packages/react-components/react-charts-preview/src/components/DonutChart/DonutChart.types.ts

```diff
@@ -1,8 +1,8 @@
-import { IStyle } from '@fluentui/react/lib/Styling';
-import { IRenderFunction, IStyleFunctionOrObject } from '@fluentui/react/lib/Utilities';
+import { IRenderFunction } from '@fluentui/react/lib/Utilities';
```

Comment:
remove IRenderFunction and StringAxis imports.

Reply:
done